experimental/oak_baremetal_app_crosvm/layout.ld

```diff
@@ -63,7 +63,7 @@ SECTIONS {
 
     /* Stack grows down, so stack_start is the upper address in memory. */
     .stack (NOLOAD) : ALIGN(4K) {
-        . += 128K;
+        . += 512K;
```

Comment:
You probably need to do the same in https://github.com/project-oak/oak/blob/main/third_party/rust-hypervisor-firmware-boot/layout.ld#L43 for the QEMU version.

Reply:
yup, thanks!